spark/processing/3.1/Pipfile

```diff
@@ -0,0 +1,38 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+tenacity = "==5.1.4"
+psutil = "==5.7.0"
+click = "==8.0.0"
+watchdog = "==0.10.3"
+waitress = "==2.1.1"
+requests = "==2.24.0"
+rsa = "==4.3"
```

Comment:
From online discussion: probably need to remove these as they are not being used

Reply:
they are used in smspark